modules/utxo-lib/src/bitgo/wallet/Unspent.ts

```diff
@@ -36,6 +36,8 @@ import { parseSignatureScript } from '../parseInput';
 import { checkForInput } from 'bip174/src/lib/utils';
 import { ProprietaryKeySubtype, PSBT_PROPRIETARY_IDENTIFIER } from '../PsbtUtil';
 
+export const DEFAULT_TX_INPUT_SEQUENCE_NUMBER = 0xffffffff;
```

Comment:
```suggestion
/** Final (non-replaceable) */
export const TX_INPUT_SEQUENCE_NUMBER_FINAL = 0xffffffff;
```

whether it is default or not is defined by how it is used